Fundamentals/Setup/Server-Setup/runtime-modes.md

```diff
@@ -133,7 +136,7 @@ public class DisableElectionForSingleServerValidator : IRuntimeModeValidator
 
     public bool Validate(RuntimeMode runtimeMode, [NotNullWhen(false)] out string? validationErrorMessage)
     {
-        if (_globalSettings.CurrentValue.DisableElectionForSingleServer == false)
+        if (runtimeMode == RuntimeMode.Production && _globalSettings.CurrentValue.DisableElectionForSingleServer == false)
```

Comment:
I've also added this additional statement, so we actually use this parameter and because it also makes more sense to only check this when using production mode 👍🏻